### DISCUSSION TITLE
繁体维护 - 是否需要帮助？

### DISCUSSION THREAD

**Discussion by N071** [SmallYuanSY/LocalizeLimbusCompany_TW#2] [Ideas]
首先，我们非常感谢您能够采用我们的模组，并以支持繁体的形式为我们做出了巨大的贡献。
——一直以来辛苦了。

我发现以单人形式维护繁体的工作量也许有些多……虽然不知道效果将会如何，但我想我们可以在B站以官号名义号召一下繁体社区的玩家来参与这边的维护。
您也可以试着在巴哈或者其他地方随意地发布并宣传您的模组，寻找别人的帮助！

**Comment by SmallYuanSY** [comment]
我目前維護的狀態會導致更新數量較多 但是目前更新都還算OK 只怕自身在忙其他東西時會跟不上更新
我已經在巴哈上面有一個心得的討論串 (https://forum.gamer.com.tw/C.php?bsn=73052&snA=297&tnum=15)
照著你們更新的進度大概比你們晚一天而已 owo>
更新上如果有問題的話 我比較希望能夠有人告知我哪邊出現問題 我馬上去修正就好了>x<

**Comment by SmallYuanSY** [comment]
大大 我想問一下 B站的圖片能夠直接拿來使用嗎? 可能會直接拿更新公告與角色技能拆包

**Reply by N071** [reply]
如果您是指我们对公告的翻译的话，那么没问题！

都市零协会授权SmallYuanSY对其在BiliBili发布的月亮计划公告翻译的专栏文章进行转载，无论BiliBili专栏文章中显示是否授权转载。
2023/9/5

若您需要披露授权情况，以此为准即可。

**Reply by SmallYuanSY** [reply]
感謝大大們

**Comment by SmallYuanSY** [comment]
有問題可以直接在這邊問大大嗎 今天在搞workflow時有些問題

**Reply by N071** [reply]
我并不是模组的主要开发者，因此我能提供的帮助有限。不过我确信我们的主要开发者能帮您解决问题！
若要联系他们还请移步我们的[旧Discussion](https://github.com/orgs/LocalizeLimbusCompany/discussions/113)

**Reply by SmallYuanSY** [reply]
OKOK owo/
